internal/pkg/manifest/validate.go

```diff
@@ -1469,6 +1484,30 @@ func (q SQSQueue) validate() error {
 	return nil
 }
 
+// validateFIFO returns nil if FIFO SQSQueue is configured correctly.
+func (q SQSQueue) validateFIFO() error {
+	if q.IsEmpty() {
+		return nil
+	}
+
+	if q.HighThroughputFifo != nil && (q.FifoThroughputLimit != nil || q.DeduplicationScope != nil) {
+		return fmt.Errorf(`validate "high_throughput_fifo": cannot define fifo_throughput_limit and fifo_throughput_limit when high_throughput_fifo is defined`)
```

Comment:
nit:
```suggestion
		return fmt.Errorf(`validate "high_throughput_fifo": cannot define "fifo_throughput_limit" and "fifo_throughput_limit" when "high_throughput_fifo" is defined`)
```

Reply:
Should we accept the values of `messageGroup` and `perMessageGroupId` even if the user sets `high_throughput_fifo` to `true`?

Reply:
We can accept the values, but then we need to again have one more condition to prioritize high_throughput_fifo over those 2 fields. So it's better error out saying you can either set high_throughput_fifo or those 2 fields.